active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,150 @@
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+
+![Breadcrumb crumbling with 3 visible nodes](images/Breadcrumb_crumbling.PNG)
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API. In V1 Breadcrumb no chevrons are interactable to show children. V2 Breadcrumb will add functionality to have flyouts from chevrons for every node, to view children of that node. This spec focuses on V1 Breadcrumb, but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Create a Breadcrumb
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
+~~~~
+More examples to be added
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Note that V2 of Breadcrumb will add flyouts from chevrons to view children of a node. Additional properties will be added to the API to enable this functionality. 
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+## API
+~~~~
+
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
```

Comment:
Why is this here? How does an app use it?

Reply:
This is a primitive used by the control, the app does not need to know about this layout.

Reply:
So we could remove this API (doesn't need to be public)?

Reply:
We could make it internal and set it from code.

Reply:
See Alberto's PR to update the API here https://github.com/microsoft/microsoft-ui-xaml-specs/pull/111/files